docs/reference/rockcraft.yaml.rst

```diff
@@ -159,6 +159,22 @@ when they fail. It uses Pebble's layer specification syntax, with each
 entry corresponding to a check. Each check can be one of three types:
 ``http``, ``tcp`` or ``exec``.
 
+``entrypoint-service``
+------------------------
+
+**Type**: string
+
+**Required**: No
+
+Makes Rockcraft extend the default OCI entrypoint
+``["/bin/pebble", "enter", "--verbose"]`` with ``["--args", "<value>"]``.
+This is used to set one of the defined Pebble services as a part of the OCI
+entrypoint.
+This option must only be used in cases where the targeted deployment
+environment has unalterable assumptions about the container image's
+entrypoint.
```

Comment:
this should be a Warning block. See an example in https://github.com/canonical/rockcraft/pull/410/files#diff-d88372eb092fe5a91cf204e108451fe4e9f34927f627138837af6b9083c1fb3bL175